Tests/test_file_cur.py

```diff
@@ -1,30 +1,133 @@
+from io import BytesIO
+
 import pytest
 
 from PIL import CurImagePlugin, Image
 
-TEST_FILE = "Tests/images/deerstalker.cur"
-
 
-def test_sanity():
-    with Image.open(TEST_FILE) as im:
+def test_deerstalker():
+    with Image.open("Tests/images/cur/deerstalker.cur") as im:
```

Comment:
Any particular reason for this change?

Reply:
I moved all the .cur files into a folder so that the Images folder was not cluttered by the ones I planned to add. At the same time I decided to rename the test since I was about to test a few more cursors (from windows 98 and up)